### PR TITLE
Deduplication module

### PR DESCRIPTION
**Pull request by jerrinot**
PR closed by Hazelcast automation as no activity (>3 months). Please reopen with comments, if necessary. Thank you for using Hazelcast and your valuable contributions

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,7 +46,7 @@ subprojects {
     targetCompatibility = '1.8'
 
     ext {
-        jetVersion = '3.1'
+        jetVersion = '3.2'
```
Comment (jerrinot) [inline build.gradle]: I am assuming this is fine as soon as 3.2 is released.

**Comment by jerrinot** [comment]
run-lab-run

**Comment by jerrinot** [comment]
verify

File: build.gradle (hunk 1 above)
Comment (gurbuzali) [reply build.gradle]: 👍

**Review by gurbuzali — COMMENTED** [review]
overall very good, just a minor comment: we usually add a very simple usage example to the javadoc

File: deduplication/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+version = 0.2-SNAPSHOT
```
Comment (eminn) [inline deduplication/gradle.properties]: It should be 0.1-SNAPSHOT

**Pull request closed without merging**